### PR TITLE
Update Manager module correctly updated

### PR DESCRIPTION
**Pull request by lloydlimmsft**

### CODE REVIEW

**Review by sebassem — CHANGES_REQUESTED** [review]
Look great LLoyd , one minor comment is if you can add a section for reporting ?

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 1]
```diff
@@ -627,10 +627,67 @@ if ($roleDefinitionIds.Count -gt 0)
...
-#### Task 2
+This information is taken from the link: https://azure.microsoft.com/en-us/products/azure-update-management-center
+
+This new service is currently in preview and has no powershell scripting option (as of Aug 2023)
+
+#### Prerequisites:
+Ensure that you already have your Jumpbox with VMs onboarded via Azure Arc.
+
+#### Onboarding the VMs
+Note that all Azure VMs and Arc Server VMs are automatically enrolled in this Azure Update Management service.
```
[the remark this answers is not in the document]
Comment (lloydlimmsft) [reply docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:644]: No, the update manager automatically sees all Azure VMs and Arc VMs.
If you go to the portal, you can see your own jumpbox VMs in the update manager, even though you never did any enrollment other than Arc.

**Comment by lloydlimmsft** [comment]
> Look great LLoyd , one minor comment is if you can add a section for reporting ?

Added.

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sebassem) [reply docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:644]: Aaaah I misunderstood, it definitely seems them but they are not onboarded (they do not get updated automatically) , you would need to create a maintenance configuration and include those vms and arc-enabled servers. So I think for the sake of clarity we change the "automatically enrolled" wording , what do you think ?

**Comment by lloydlimmsft** [comment]
Hi Seif,

I can use the words "already visible".

Thank you,
Lloyd Lim.
Call me at +6493625625<tel:+6493625625>. It rings my mobile and desk phone simultaneously.

**Pull request merged**